docs/chapters/workflow/release/release-process.rst

```diff
@@ -9,10 +9,36 @@ There are two types of releases, source releases, and binary releases. A binary
 release has all the requirements of a source release and more. This split is for
 licensing reasons, as there are more obligations when doing a binary release.
 
+Binary release
+--------------
+Binaries of pelux are released on each process and can be found under our `downloads page`_.
+
+We currently release 4 different variants (most with 2 images each).
+
+.. This is to get red and green colours for the symbols below
+.. role:: available
+.. role:: unavailable
+.. raw:: html
+
+    <!-- The roles we defined in rst will translate to style sheet classes -->
+    <style> .available {color:green} .unavailable {color:red} </style>
+    
++--------------------------------------------+------------------+------------------+------------------+-------------------+
+|                                            |      Variant name                                                          |
++          Image name                        +------------------+------------------+------------------+-------------------+
+|                                            | intel            | arp              | rpi              | qemu              |
++============================================+==================+==================+==================+===================+
+| core-image-pelux-minimal-dev               | :available:`✔`   | :available:`✔`   | :available:`✔`   | :available:`✔`    |
++--------------------------------------------+------------------+------------------+------------------+-------------------+
+| core-image-pelux-qtauto-neptune-dev        | :available:`✔`   | :available:`✔`   | :available:`✔`   | :unavailable:`✘`  |
++--------------------------------------------+------------------+------------------+------------------+-------------------+ 
+
+
 Source release
 --------------
-In a source release, we release no binaries, which means that any binary parts,
-such as the platform itself, will have to be built by the user.
+The release process of PELUX sources is compliant to yocto source release which can be found 
+on `Yocto Development Manual`_. The URL for PELUXs' released sources is here_.
```

Comment:
A manifest file (.xml) pointing out all layers used in a Yocto build of the PELUX baseline (would typically be pelux.xml in the pelux-manifests repository).

Reply:
@AshleyNair The manifest file is actually not released but it can be found on github. Therefore i wont mention it under "Source release"

Reply:
What does it mean to be compliance with Yocto's source release?

Reply:
On the given link, there is a guide on how to release and zip the sources. Thats what i was referring to by saying 'compliant with yocto's release.

Reply:
@fhajredini I think that part of the sentence might be a bit unclear. Do you mean "compliant with the Yocto source release process" or "compliant with [a certain] Yocto release". Maybe that's what @jeremiah means.

Reply:
I see. Yeah it might be a bit unclear, but i was referring to the process that Yocto offers guidelines for.

Reply:
For me, the word 'compliant' was confusing because that word is commonly used with licenses, like "compliant with the GPL" or similar. And Yocto doesn't require any compliance but it does have a set of best practices for managing  source code. So I understand what you're saying @fhajredini and I'm sorry for nit-picking but I was confused since I thought we were referring to licenses here.

Reply:
I think it's important to be clear in the documentation, so I think this part should be updated to avoid confusion. Thanks, @jeremiah.

Reply:
Agreed.

Reply:
I made an issue for this here on GitHub: #155